src/test/suite/fixtures/recorded/compoundTargets/clearThisSliceBatt.yml

```diff
@@ -0,0 +1,38 @@
+languageId: plaintext
+command:
+  spokenForm: clear this slice batt
+  version: 3
+  targets:
+    - type: range
+      anchor:
+        type: primitive
+        mark: {type: cursor}
+      active:
+        type: primitive
+        mark: {type: decoratedSymbol, symbolColor: default, character: b}
+      excludeAnchor: false
+      excludeActive: false
+      rangeType: vertical
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |-
+    aa
+    bb
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 1}
+  marks:
+    default.b:
+      start: {line: 1, character: 0}
+      end: {line: 1, character: 2}
+finalState:
+  documentContents: |-
+    a
+    b
```

Comment:
huh.  is this what we want?  seems inconsistent

Reply:
The selection is non empty and therefore does not expand